modules/dataquery/jsx/react.app.js

```diff
@@ -440,6 +440,26 @@ var DataQueryApp = React.createClass({
                 loading: false
             }
         });
+        for (var i = 0; i < fieldsList.length; i++) {
+            var that = this;
```

Comment:
@jacobpenny Yes this can but this code is not using es6 syntax and is staying consistent with the format of the existing code